src/components/cta/ExternalsAuth.tsx

```diff
@@ -102,34 +133,63 @@ export const ExternalsAuth: React.FunctionComponent<ExternalsAuthProps> = ({
         })
     }
 
-    return authProvider === 'github' ? (
-        <Link
-            href="https://sourcegraph.com/.auth/github/login?pc=https%3A%2F%2Fgithub.com%2F%3A%3Ae917b2b7fa9040e1edd4&redirect=/get-cody"
-            className={classNames(
-                `btn hover:sg-bg-hover-external-auth-button flex items-center px-4 hover:text-black md:h-12 md:px-6 md:text-lg
+    let link: React.ReactElement;
+    link =  <Link href="" />;
```

Comment:
Nit: We could do this more succinctly, and only one space after =
```suggestion
    let link: React.ReactElement = <Link href="" />;
```